From cb23decd8f4b4c09ed0a6f568d4248aa36e77481 Mon Sep 17 00:00:00 2001
From: Pawit Pornkitprasan <p.pawit@gmail.com>
Date: Sat, 2 Nov 2013 23:02:15 -0700
Subject: [PATCH] libEGL: allow devices to workaround Google bug 10194508

Most older blobs don't have the EGL_KHR_gl_colorspace
extension and need this workaround.

Change-Id: Ifc5d097c1a0b68275d23c2d191976ac96631b695
---
 opengl/libs/Android.mk | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/opengl/libs/Android.mk b/opengl/libs/Android.mk
index cc5d544..4838d6c 100644
--- a/opengl/libs/Android.mk
+++ b/opengl/libs/Android.mk
@@ -50,6 +50,9 @@ endif
 ifeq ($(TARGET_BOARD_PLATFORM), omap4)
   LOCAL_CFLAGS += -DWORKAROUND_BUG_10194508=1
 endif
+ifeq ($(BOARD_EGL_WORKAROUND_BUG_10194508),true)
+  LOCAL_CFLAGS += -DWORKAROUND_BUG_10194508=1
+endif
 ifneq ($(MAX_EGL_CACHE_ENTRY_SIZE),)
   LOCAL_CFLAGS += -DMAX_EGL_CACHE_ENTRY_SIZE=$(MAX_EGL_CACHE_ENTRY_SIZE)
 endif
-- 
1.9.1

